src/styles/_catalog.scss

```diff
@@ -14,6 +14,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: 300ms;
```

Comment:
You don't need to set `transition` for all animations, only for transform and color

Reply:
fixed